src/games/strategy/triplea/util/TransportUtils.java

```diff
@@ -178,7 +176,7 @@ public int compare(final Unit o1, final Unit o2) {
       public int compare(final Unit o1, final Unit o2) {
         final int cost1 = UnitAttachment.get((o1).getUnitType()).getTransportCost();
         final int cost2 = UnitAttachment.get((o2).getUnitType()).getTransportCost();
-        return cost2 - cost1;
+        return Integer.compare(cost2, cost1);
```

Comment:
Again looks like ordering might be swapped. When cost2 is larger than cost1, before we had a positive number. Now with the compare, we'll get a negative number.